cluster-operator/src/main/resources/cluster-roles/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -19,6 +19,8 @@ rules:
   - apiGroups:
       - "rbac.authorization.k8s.io"
     resources:
+      # The cluster operator needs to access and manage roles to grant the entity operator permissions
```

Comment:
Presumably we only need this in role (not cluster role) mode? If so, we should say so in the comment.